src/test/java/com/google/cloud/spanner/pgadapter/PgxMockServerTest.java

```diff
@@ -130,15 +140,16 @@ public void testHelloWorld() {
 
     assertNull(res);
     List<ExecuteSqlRequest> requests = mockSpanner.getRequestsOfType(ExecuteSqlRequest.class);
-    // pgx sends the query three times:
-    // 1. DESCRIBE statement
-    // 2. DESCRIBE portal
-    // 3. EXECUTE portal
-    assertEquals(3, requests.size());
+    // The patched version of pgx sends the query two times.
+    // This will be one time when https://github.com/GoogleCloudPlatform/pgadapter/pull/80 has been
+    // merged.
+    // 1. DESCRIBE portal
+    // 2. EXECUTE portal
+    assertEquals(2, requests.size());
```

Comment:
Yay! One less round trip :-)

And when #80 is merged, it will be down to just one round trip.